doc/classes/Dictionary.xml

```diff
@@ -340,6 +340,23 @@
 				[b]Note:[/b] [method merge] is [i]not[/i] recursive. Nested dictionaries are considered as keys that can be overwritten or not depending on the value of [param overwrite], but they will never be merged together.
 			</description>
 		</method>
+		<method name="merged" qualifiers="const">
+			<return type="Dictionary" />
+			<param index="0" name="dictionary" type="Dictionary" />
+			<param index="1" name="overwrite" type="bool" default="false" />
+			<description>
+				Returns a copy of this dictionary merged with the other [param dictionary]. Useful for quickly making dictionaries with default values.
```

Comment:
The small description of `overwrite` above sounds good enough. `merge()` could be pointed out sooner instead of being all the way below.
```suggestion
				Returns a copy of this dictionary merged with the other [param dictionary]. By default, duplicate keys are not copied over, unless [param overwrite] is [code]true[/code]. See also [method merge].
				This method is useful for quickly making dictionaries with default values:
```